### PR TITLE
Added permanent include statement for custom Monit files

### PR DESCRIPTION
**Pull request by codiflow**
Referencing https://github.com/opnsense/core/issues/6337

### CODE REVIEW

**Comment by codiflow** [comment]
I checked the paths twice (in PR and in docs) and `/usr/local/etc/monit.opnsense.d` with monit.**opnsense**.d should be fine.

We could also use monit.**conf**.d but I think having the opnsense here is not a problem at all 😎

File: src/opnsense/service/templates/OPNsense/Monit/+TARGETS [hunk 1]
```diff
@@ -1,2 +1,3 @@
 monitrc:/usr/local/etc/monitrc
 rc.conf.d:/etc/rc.conf.d/monit
+00-example.conf:/usr/local/etc/monit.opnsense.d/00-example.conf
```
Comment (fichtner) [inline src/opnsense/service/templates/OPNsense/Monit/+TARGETS:3]: Can we do this without the templates? Just add the proper git file directly

Comment (codiflow) [reply src/opnsense/service/templates/OPNsense/Monit/+TARGETS:3]: I'm sorry but I don't get what you mean. Could you explain me what exactly are you expecting and where should this file be located then?

Comment (fichtner) [reply src/opnsense/service/templates/OPNsense/Monit/+TARGETS:3]: remove this entry and move 00-example.conf to src/etc/monit.opnsense.d/00-example.conf and done

**Comment by codiflow** [comment]
Would appreciate if you could check again if the PR is now ready for being merged. Thanks!

File: src/etc/monit.opnsense.d/00-example.conf [hunk 2]
```diff
@@ -0,0 +1 @@
+# This is a placeholder file. You can add your custom and advanced Monit configuration here.
```
Comment (fichtner) [inline src/etc/monit.opnsense.d/00-example.conf:1]:
```suggestion
# This is a placeholder file. You can add your advanced Monit configuration in this directory.
```

**Comment by fichtner** [comment]
I suppose @AdSchellevis agrees... looks simple enough now, thanks!

**Pull request merged**